### DISCUSSION TITLE
Separate options "lock when closed screen" and "(Windows) User Session is locked"

### DISCUSSION THREAD

**Discussion by Ilutio** [keepassxreboot/keepassxc#7017] [Ideas]
I have the issue that, in Settings->Security, the option: Lock database when screen is closed or Windows User Session is locked.
these options are not seprated.
"closed screen" and "locked windows session" should be separated. I close the screen several times a day because I have to go to Meetings, and then the next Meeting etc. etc. Please solve this issue.

**Comment by phoerious** [comment]
What speaks against simply locking the screen like any sane person would do? Usually, screen lock and lid close are connected.

**Reply by droidmonkey** [reply]
Technically we can do this, but I cant even configure windows 10 to NOT lock when the lid is closed. Maybe that is because my computer goes to sleep when that happens too.

**Reply by Ilutio** [reply]
I use Windows 10 and lid close is NOT connected with workspace lock. It might be that you can activate also workspace lock when you close the lid but please let users like me the option open. It doesn't limit you in any way.
I just want to enter a meeting, open the lid and work like nothing happened.

**Comment by Thomas-MB** [comment]
Same issue here. I'm using normally only external monitors and configured my laptop in the way, that it doesn't goes into standby when screen closing.
Lastly I solved this with unlocking after closing my laptop screen. But now, there is the nice new keepassxc feature to unlock with fingerprint. Unfortunately, I have to open my laptop screen, then can finger print unlock and after closing my screen again, the db is closed again.

Currently I, disable the auto lock feature, which is not very safe. To explicitly remember to lock the db by myself is a hard thing ;).

So, please separate the 2 auto lock features.

Hint: To configure your laptop not going to standby, you have to go into energy options.
